android-project/app/src/main/java/org/libsdl/app/SDLSurface.java

```diff
@@ -214,6 +222,13 @@ private float getNormalizedY(float y)
     // Touch events
     @Override
     public boolean onTouch(View v, MotionEvent event) {
+
+        if (scaleGestureDetector.onTouchEvent(event)) {
+            if (scaleGestureDetector.isInProgress()) {
+                return true; /* event was consumed as gesture */
```

Comment:
We probably need a way to disable this for applications that want touch and don't care about zoom.